source/context.h

```diff
@@ -1,40 +1,20 @@
 #ifndef _CONTEXT_H
 #define _CONTEXT_H
 
-#include <libavformat/avformat.h>
 #include <switch.h>
-#include <SDL.h>
-#include <SDL2/SDL_image.h>
-#include "SDL_FontCache.h"
```

Comment:
this results in a compiler-error on linux

`#include "sdl_fontcache.h"` seems to work